utils/docker.go

```diff
@@ -322,7 +322,38 @@ func GetPFEHostAndPort() (string, string) {
 			}
 		}
 	}
-	return "",""
+	return "", ""
+}
+
+// GetImageTag of Codewind images
+func GetImageTag() []string {
```

Comment:
This should be plural right?

```suggestion
func GetImageTags() []string {
```